### PR TITLE
[new release] json_decoder (v0.1.1)

### PR DESCRIPTION
**Pull request by dagoof**
JSON Decoder based on Elm's Json.Decode

- Project page: <a href="https://github.com/dagoof/ocaml-json-decoder">https://github.com/dagoof/ocaml-json-decoder</a>
- Documentation: <a href="https://dagoof.github.io/ocaml-json-decoder/doc">https://dagoof.github.io/ocaml-json-decoder/doc</a>

##### CHANGES:

- update to opam v2

### CODE REVIEW

File: packages/json_decoder/json_decoder.0.1.1/descr [hunk 1]
```diff
@@ -0,0 +1,8 @@
+JSON Decoder based on Elm's Json.Decode
+Release %%VERSION%%
```
Comment (mseri) [inline packages/json_decoder/json_decoder.0.1.1/descr]: Does this need to be substituted?

File: packages/json_decoder/json_decoder.0.1.1/url [hunk 2]
```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/dagoof/ocaml-json-decoder/releases/download/v0.1.1/json_decoder-0.1.1.tbz"
+checksum: "04d5c7b9b55ec973bb88ac55789addef"
```
[the remark this answers is not in the document]
Comment (dagoof) [reply packages/json_decoder/json_decoder.0.1.1/url]: Thank you, will fix.

File: packages/json_decoder/json_decoder.0.1.1/descr (hunk 1 above)
Comment (dagoof) [reply packages/json_decoder/json_decoder.0.1.1/descr]: Probably. New to this but will get it sorted

**Comment by mseri** [comment]
Thanks for the patience!

**Pull request merged**